tests/test_utils.py

```diff
@@ -105,6 +105,15 @@ def test_assert_redirects_custom_message(self):
         except AssertionError as e:
             self.assertTrue("Custom message" in str(e))
 
+    def test_assert_redirects_valid_status_codes(self):
```

Comment:
Probably worth having a redirect that fails now that the route has the ability to choose the code via query string args